### ISSUE TITLE
mail.ru on 32-bit ARM: cannot unmarshal number 111751017437 into Go struct field .body.cloud.space.bytes_used of type int

### ISSUE THREAD

**Issue by jezovuk** [rclone/rclone#6003]
#### What is the problem you are having with rclone?

Command `about` for cloud@mail.ru account fails with following error:
```
Failed to about: About call failed: json: cannot unmarshal number 111751017437 into Go struct field .body.cloud.space.bytes_used of type int
```

#### What is your rclone version (output from `rclone version`)
```
rclone v1.57.0
- os/version: unknown
- os/kernel: 4.2.8 (armv7l)
- os/type: linux
- os/arch: arm
- go/version: go1.17.2
- go/linking: static
- go/tags: none
```

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

QNAP NAS TS-231P  QTS 5.0.0.1932

Note that this is a 32-bit ARM system:

CPU - AnnapurnaLabs, an Amazon company Alpine AL212, 2-core, 1.7GHz
CPU Architecture - 32-bit ARM

#### Which cloud storage system are you using? (e.g. Google Drive)

cloud@mail.ru

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

`sudo rclone about mailru: --config /share/CACHEDEV1_DATA/.qpkg/RClone/.config/rclone/rclone.conf`

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

```
2022/02/20 02:11:42 DEBUG : rclone: Version "v1.57.0" starting with parameters ["rclone" "-vv" "about" "mailru:" "--config" "/share/CACHEDEV1_DATA/.qpkg/RClone/.config/rclone/rclone.conf"]
2022/02/20 02:11:42 DEBUG : Creating backend with remote "mailru:"
2022/02/20 02:11:42 DEBUG : Using config file from "/share/CACHEDEV1_DATA/.qpkg/RClone/.config/rclone/rclone.conf"
2022/02/20 02:11:42 DEBUG : 4 go routines active
2022/02/20 02:11:42 Failed to about: About call failed: json: cannot unmarshal number 111751017437 into Go struct field .body.cloud.space.bytes_used of type int
```

#### Suggestion

From quick code inspection it seems that the problem is in https://github.com/rclone/rclone/blob/v1.57.0/backend/mailru/api/m1.go#L83. Here, type of `BytesUsed` is `int`, while it seems it should be `int64`. (Note that `BytesTotal` is `int64`). If I understand correctly (not fluent in Golang), `int` is platform dependent and probably defautls to (signed) 32-bit on my NAS ARM 32-bit CPU. Since I have ~110GB of data stored in mail.ru cloud, it overflows signed 32-bit int.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ivandeex** [comment]
Please try this beta
https://beta.rclone.org/branch/iva-fix-mailru-arm32/v1.58.0-beta.6001.03eab4233.iva-fix-mailru-arm32/

**Comment by ivandeex** [comment]
@jezovuk 
Please try this beta
https://beta.rclone.org/branch/iva-fix-mailru-arm32/v1.58.0-beta.6001.03eab4233.iva-fix-mailru-arm32/

**Comment by jezovuk** [comment]
Works like a charm :+1:

**Comment by ivandeex** [comment]
Good.
The patch will be merged on trunk and released by 1.60

**Comment by ncw** [comment]
@ivandeex is this a small fix? If so we could squeeze it into 1.59?

**Comment by ncw** [comment]
Mmm, headlines and deadlines... I'd like to do the very delayed 1.60 soon (week or two) - I'll make an issue about it... So enough time to squeeze a small fix in.

**Comment by ivandeex** [comment]
> very delayed 1.59 soon (week or two)

1.58 you mean ;)

**Comment by ncw** [comment]
I've merged this to fix master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.58 - thank you @ivandeex

**Comment by jCrompton** [comment]
How was this resolved exactly? Think I am dealing with a similar issue but unrelated to this codebase
